### PR TITLE
Fix #11378: Sign setup.exe

### PR DESCRIPTION
**Pull request by jumde**
Resolves https://github.com/brave/brave-browser/issues/11378

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [x] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [x] Windows
- [ ] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [ ] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [ ] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [ ] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Test Plan:
```
$ cd  C:\Program Files (x86)\Windows Kits\10\bin\10.0.19041.0\x64

$ .\signtool.exe verify /pa "C:\Program Files\BraveSoftware\Brave-Browser-Nightly\Application\<cr_version>\Installer\setup.exe"
File: C:\Program Files\BraveSoftware\Brave-Browser-Nightly\Application\85.1.15.2\Installer\setup.exe
Index  Algorithm  Timestamp
========================================
0      sha256     Authenticode

Successfully verified: C:\Program Files\BraveSoftware\Brave-Browser-Nightly\Application\<cr_version>\Installer\setup.exe

$ .\signtool.exe verify /pa "C:\Program Files\BraveSoftware\Brave-Browser-Nightly\Application\<cr_version>\chrome.dll"
File: C:\Program Files\BraveSoftware\Brave-Browser-Nightly\Application\85.1.15.2\chrome.dll
Index  Algorithm  Timestamp
========================================
0      sha256     Authenticode

Successfully verified: C:\Program Files\BraveSoftware\Brave-Browser-Nightly\Application\85.1.15.2\chrome.dll

$ .\signtool.exe verify /pa "C:\Program Files\BraveSoftware\Brave-Browser-Nightly\Application\brave.exe"
File: C:\Program Files\BraveSoftware\Brave-Browser-Nightly\Application\brave.exe
Index  Algorithm  Timestamp
========================================
0      sha256     Authenticode

Successfully verified: C:\Program Files\BraveSoftware\Brave-Browser-Nightly\Application\brave.exe
```

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

File: script/create_installer_archive_helper.py [hunk 1]
```diff
@@ -24,6 +23,9 @@ def SignAndCopyPreSignedBinaries(skip_signing, output_dir, staging_dir, current_
         shutil.copy(os.path.join(src_dir, 'brave.exe'), chrome_dir)
         shutil.copy(os.path.join(src_dir, 'chrome.dll'), version_dir)
 
+def SignSetupExe(skip_signing, output_dir):
```
Comment (simonhong) [inline script/create_installer_archive_helper.py:26]: nit: expected two blank lines above - https://travis-ci.org/github/brave/brave-core/jobs/720446370

**Review by simonhong — APPROVED** [review]
++ and nits with py lint error.

**Review by mihaiplesa — APPROVED** [review]

**Comment by jumde** [comment]
CI failures are known.

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Win 10 x64` using the following build:

```
Brave | 1.15.23 Chromium: 85.0.4183.83 (Official Build) nightly (64-bit)
--- | ---
Revision | 94abc2237ae0c9a4cb5f035431c8adfb94324633-refs/branch-heads/4183@{#1658}
OS | Windows 10 OS Version 2004 (Build 19041.450)
```

```
PS C:\Program Files (x86)\Windows Kits\10\bin\10.0.18362.0\x64> .\signtool.exe verify /pa "C:\Program Files\BraveSoftware\Brave-Browser-Nightly\Application\85.1.15.23\Installer\setup.exe"
File: C:\Program Files\BraveSoftware\Brave-Browser-Nightly\Application\85.1.15.23\Installer\setup.exe
Index  Algorithm  Timestamp
========================================
0      sha256     Authenticode

Successfully verified: C:\Program Files\BraveSoftware\Brave-Browser-Nightly\Application\85.1.15.23\Installer\setup.exe
```

Annotation 2020-08-31 173050


```
PS C:\Program Files (x86)\Windows Kits\10\bin\10.0.18362.0\x64> .\signtool.exe verify /pa "C:\Program Files\BraveSoftware\Brave-Browser-Nightly\Application\85.1.15.23\chrome.dll"
File: C:\Program Files\BraveSoftware\Brave-Browser-Nightly\Application\85.1.15.23\chrome.dll
Index  Algorithm  Timestamp
========================================
0      sha256     Authenticode

Successfully verified: C:\Program Files\BraveSoftware\Brave-Browser-Nightly\Application\85.1.15.23\chrome.dll
```

Annotation 2020-08-31 173149

```
PS C:\Program Files (x86)\Windows Kits\10\bin\10.0.18362.0\x64> .\signtool.exe verify /pa "C:\Program Files\BraveSoftware\Brave-Browser-Nightly\Application\brave.exe"
File: C:\Program Files\BraveSoftware\Brave-Browser-Nightly\Application\brave.exe
Index  Algorithm  Timestamp
========================================
0      sha256     Authenticode

Successfully verified: C:\Program Files\BraveSoftware\Brave-Browser-Nightly\Application\brave.exe
```

Annotation 2020-08-31 173648

**Comment by GeetaSarvadnya** [comment]
Verification passed on **Windows 10 x64** - using

```

Brave | 1.15.25 Chromium: 85.0.4183.83 (Official Build) nightly (64-bit)
-- | --
Revision | 94abc2237ae0c9a4cb5f035431c8adfb94324633-refs/branch-heads/4183@{#1658}
OS | Windows 10 OS Version 1903 (Build 18362.1016)

```
```

c:\Program Files (x86)\Windows Kits\10\bin\10.0.19041.0\x64>bravesign.exe verify /pa "C:\Program Files\BraveSoftware\Brave-Browser-Nightly\Application\85.1.15.25\Installer\setup.exe"
File: C:\Program Files\BraveSoftware\Brave-Browser-Nightly\Application\85.1.15.25\Installer\setup.exe
Index  Algorithm  Timestamp
========================================
0      sha256     Authenticode

Successfully verified: C:\Program Files\BraveSoftware\Brave-Browser-Nightly\Application\85.1.15.25\Installer\setup.exe
```

**Comment by btlechowski** [comment]
Verified passed on **Windows 7**


Brave | 1.15.23 Chromium: 85.0.4183.83 (Official Build) nightly (64-bit)
-- | --
Revision | 94abc2237ae0c9a4cb5f035431c8adfb94324633-refs/branch-heads/4183@{#1658}
OS | Windows 7 Service Pack 1 (Build 7601.24544)

image
image

image
image

image
image

**Pull request merged**